src/node.cc

```diff
@@ -704,6 +704,12 @@ const char *signo_string(int signo) {
 # endif
 #endif
 
+#ifdef SIGINFO
+# if SIGINFO != SIGPWR
```

Comment:
I think this should be `# if defined(SIGPWR) && SIGINFO != SIGPWR` - `SIGINFO` does not imply the existence of `SIGPWR`.